ci/generate-workflows.py

```diff
@@ -953,23 +962,26 @@ def generate_actions(namer, jobber, trigger, is_continuous, is_tag=False):
             container = ""
 
         trigger_paths = [file_name]
-        trigger_paths += TRIGGER_PATHS
-        if "win" in name:
-            trigger_paths += TRIGGER_PATHS_WIN
-        elif "macos" in name:
-            trigger_paths += TRIGGER_PATHS_MAC
-        else:
-            trigger_paths += TRIGGER_PATHS_UNIX
-        if t.app_image:
-            trigger_paths += TRIGGER_PATHS_APPIMAGE
+        if t.priorize or is_continuous or is_tag:
+            trigger_paths += TRIGGER_PATHS
+            if "win" in name:
+                trigger_paths += TRIGGER_PATHS_WIN
+            elif "macos" in name:
+                trigger_paths += TRIGGER_PATHS_MAC
+            else:
+                trigger_paths += TRIGGER_PATHS_UNIX
+            if t.app_image:
+                trigger_paths += TRIGGER_PATHS_APPIMAGE
 
         trigger_paths = "- " + "\n      - ".join(yv(p) for p in sorted(trigger_paths))
         trigger_with_paths = trigger.replace("@PATHS@", trigger_paths)
+        on_workflow = "" if t.priorize or is_continuous or is_tag else f"  workflow_run:\n    types: [completed]\n    branches: [main]\n    workflows: {yv(PRIORITY_TARGETS)}\n"
```

Comment:
I'd be happy to have the prioritization stuff working for continuous as well

Reply:
I'll try that in a bit, the fix is more involved

Reply:
Reworked the `trigger` in order to implement it.